src/components/Footer.jsx

```diff
@@ -1,10 +1,122 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Box,
+  Container,
+  Typography,
+  Grid,
+  Link as MuiLink,
+} from '@mui/material';
+import {
+  genMailToLink,
+  reportIssue,
+  featureRequest,
+  withDeviceInfo,
+} from '../lib/mail';
+import { SUBSCRIPTION_FORM_URL } from '../config/constants';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+
+const quickLinks = {
+  'footer.addScholarship': '/scholarships/new',
+  'footer.browseScholarship': '/scholarships',
+};
+
+const companyLinks = {
+  'footer.about': '/about',
+  'footer.contact': '/contact',
+};
+
+const helpLinks = {
+  'footer.reportIssue': genMailToLink({
+    subject: 'Bug Report',
+    body: withDeviceInfo(reportIssue),
+  }),
+  'footer.suggestIdea': genMailToLink({
+    subject: 'Feature Request',
+    body: withDeviceInfo(featureRequest),
+  }),
+  'footer.subscribeForUpdates': SUBSCRIPTION_FORM_URL,
+  'footer.reachOut': genMailToLink({
+    subject: 'Outreach',
+    body: 'Please describe the purpose of your outreach below.\n',
+  }),
+};
+
+function FooterColumn({ title, links, internal, t }) {
+  return (
+    <Grid item>
+      <Typography
+        color={(theme) => theme.palette.grey[500]}
+        sx={{ fontWeight: 'bold' }}>
+        {t(title)}
+      </Typography>
+      {Object.entries(links).map(([name, link]) => (
+        <Box key={name}>
+          <MuiLink
+            component={internal ? Link : MuiLink}
+            to={internal ? link : ''}
+            href={!internal ? link : ''}
+            variant="subtitle2"
+            color="text.secondary"
+            underline="hover">
+            {t(name)}
+          </MuiLink>
+        </Box>
+      ))}
+    </Grid>
+  );
+}
+
+FooterColumn.propTypes = {
+  title: PropTypes.string.isRequired,
+  links: PropTypes.objectOf(PropTypes.string).isRequired,
+  internal: PropTypes.bool,
+  t: PropTypes.func,
+};
+FooterColumn.defaultProps = {
+  internal: false,
+};
+
+function Footer(props, ref) {
+  const { t } = useTranslation();
 
-function Footer() {
   return (
-    <div>
-      <p />
-    </div>
+    <Box sx={{ bgcolor: 'background.secondary' }} ref={ref}>
+      <Container maxWidth="lg" sx={{ py: 4 }}>
+        <Grid container justifyContent="space-between" spacing={3}>
+          <Grid item xs={12} sm={6}>
```

Comment:
I'm realizing this looks strange:
<img width="866" alt="Screen Shot 2022-03-05 at 7 57 21 PM" src="https://user-images.githubusercontent.com/8023233/156908581-e62607e0-824d-4989-817d-7e38ee4a2bed.png">

Reply:
Mobile portrait also looks strange

<img width="325" alt="Screen Shot 2022-03-05 at 8 01 12 PM" src="https://user-images.githubusercontent.com/8023233/156908656-95fe2301-95df-4d62-a614-c5f18b7a1087.png">

Reply:
How about just:
```suggestion
          <Grid item>
```

Alternatively you could make all of them `sx={12}` and `sm={undefined}` (or maybe `0`?)